designs/cloudWatch/cloudWatchIntegration.md

```diff
@@ -0,0 +1,76 @@
+# CloudWatch Logs Experience
+
+* **Type**: UX Design
+* **Status**: Proposal
+
+## Introduction
+CloudWatch Logs enables a user to centralize the logs from all of their systems, applications, and AWS services that they use, in a single, highly scalable service. This document describes adding CloudWatch Logs into the JetBrains toolkit in order to facilitate easily viewing and searching them for specific error codes or patterns.
+
+## Comparison to other products
+
+| Product                       | Difference    |
+|-------------------------------|---------------|
+| AWS Toolkit for Visual Studio | Not supported |
+| AWS Toolkit for VSCode        | Not supported |
+
+## Experience
+
+### View Log Groups
+A new AWS Explorer root node would be added for the CloudWatch service. Underneath of that node we would add the list of Log Groups that exist in the selected region for the selected credential profile. Double clicking a Log Group would open the CloudWatch tool window with the tab for the Log Streams that belong to that group. A new tab would be created if needed.
```

Comment:
I'm concerned about the quantity of child nodes here. I prefer this over the alternate proposal though.
What if the root node had some sort of context menu to accomplish search (either as "starts with" or "contains" text style search). 

Possible ideas:
* prompt for a text, then filter the child nodes
* open to a separate tab which then operates as a searchable list

Reply:
So a tree node with no children?

> open to a separate tab which then operates as a searchable list

Open a tab where?

Reply:
the lists are already searchable by just typing. eg
![2020-02-20_16-10-36](https://user-images.githubusercontent.com/4661536/74991726-ac401900-53fb-11ea-8d79-b67427cf412a.gif)